lib/ibge/reader/helpers.rb

```diff
@@ -0,0 +1,63 @@
+module Ibge
+  module Reader
+    module Helpers
+      SOURCES = {
+        dtb: 'DTB_2014_subdistrito.xls',
+        sidra: 'sidra.xls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.